### DISCUSSION TITLE
How to apply same opacity attribute/property value when select multiple elements?

### DISCUSSION THREAD

**Discussion by isman198** [clientIO/joint#1963] [Q&A]
Recently, we need extend feature app based on JoinJS - Rappid toolkit, to enable select multiple elements and apply same opacity attribute/property values.  Is it possible to show input dialog box to set opacity when event select multiple events with right click?

**Comment by kumilingus** [comment]
If you want to edit multiple elements at once, here's an [example](https://codepen.io/jointjs/pen/VwMJGxQ). Is this what you were asking bout?
